docs/install.md

```diff
@@ -108,13 +108,11 @@ daemonset.apps/csi-hostpathplugin created
 
 ## Installing from the release page  (only developer preview level support)
 
-1. Run the following command
+Follow the instructions for any of the releases listed on [the Releases page](https://github.com/openshift/csi-driver-projected-resource/releases) that you are interested in, starting with release 'v0.4.8-rc.0'
```

Comment:
I don't like redirecting folks to another page in docs. In #37 I let the user pick the version via an env var.

Reply:
how do they know which value to set to the env var unless they go to the other page?